packages/buffetjs-custom/src/components/DateTime/index.js

```diff
@@ -78,7 +82,7 @@ function DateTime({
       setDate(newDate);
     }
     setIsInit(true);
-  }, []);
+  }, [setDate, value]);
```

Comment:
Adding the `value` and `setDate` makes the effect to be run multiple times when changing either the date or the value which causes multiples renders.